src/core/public/doc_links/doc_links_service.ts

```diff
@@ -403,6 +403,9 @@ export class DocLinksService {
             loadingData: `${OPENSEARCH_WEBSITE_DOCS}`,
             introduction: `${OPENSEARCH_WEBSITE_DOCS}`,
           },
+          dataSource: {
+            guide: 'https://opensearch.org/docs/latest/dashboards/discover/multi-data-sources/',
```

Comment:
can you use `${OPENSEARCH_WEBSITE_DOCS}dashboards/discover/multi-data-sources`

Reply:
OPENSEARCH_WEBSITE_DOCS appears to have branching support.
```
    const DOC_LINK_VERSION =
      injectedMetadata.getOpenSearchDashboardsBranch() === 'main'
        ? 'latest'
        : injectedMetadata.getOpenSearchDashboardsBranch();
```
 Don't think the MD doc currently have support for this.

Reply:
Can you confirm? I think everything on doc website is versioned, the same with MD

Reply:
Changed to : guide: `${OPENSEARCH_DASHBOARDS_VERSIONED_DOCS}discover/multi-data-sources/`